index.js

```diff
@@ -18,3 +18,12 @@ module.exports = function (ms) {
 
 	return thunk;
 };
+
+module.exports.reject = function (ms, value) {
```

Comment:
I meant, I think you should be able to also do:

```js
Promise.resolve(val).then(fn.reject(50, new Error('foo')));
```

In this case the error is used instead of the resolved value.

// @jamestalmage